Cargo.toml

```diff
@@ -27,3 +27,6 @@ gcc = "0.3"
 [dev-dependencies.bitcoin]
 version = "0.13"
 features = ["bitcoinconsensus"]
+
+[dev-dependencies]
+hex = "0.3.2"
```

Comment:
I'm generally fine with this, but now we've got two ways of doing hex - using rust-bitcoin's utils one, and this. It probably makes sense to convert them all to the hex crate, cause using an exported rust-bitcoin util method always felt strange to me.

Reply:
Also, not gonna block on this, but would be nice to fix in a followup PR.

Reply:
ack, I'll look into it after I wrap up https://github.com/rust-bitcoin/rust-bitcoin/pull/107